specification/summary-care-record.yaml

```diff
@@ -812,4 +810,4 @@ components:
       required: true
       schema:
         type: string
-        example: 'application/fhir+json'
+        example: "application/fhir+json"
```

Comment:
Has changing the quotes fixed any problems? If not, maybe revert this file, because the changes don't seem relevant, and leaves things a bit inconsistent